CentralityiOS/CentralityiOS/ViewControllers/ToDoFeedViewController.m

```diff
@@ -16,13 +16,17 @@
 #import "SuggestionAlgorithm.h"
 
 @interface ToDoFeedViewController () <UITableViewDelegate, UITableViewDataSource>
+@property (weak, nonatomic) IBOutlet UITabBarItem *tabButton;
 @property (weak, nonatomic) IBOutlet UIButton *logoutButton;
 @end
 
+NSTimer* notificationTimer;
```

Comment:
This should probably be inside of `ToDoFeedViewController` as a member variable. Let me know if that's causing any issues.